packages/moonstone/styles/internal/fonts.less

```diff
@@ -47,16 +47,29 @@
 	ta: local("LG Smart UI Tamil");
 	te: local("LG Smart UI Telugu");
 	ur: local("LG Smart UI Urdu");
-	zh-HK: local("LG Smart UI TC");
-	zh-TW: local("LG Smart UI TC"); // same as `zh-HK`
+	zh-HK: local("LG Smart UI TC"), local("LG Smart UI TC") 300, local("LG Smart UI TC") 600, local("LG Smart UI TC") 700;
```

Comment:
is `zh` the only locale that should use the same fonts for all weights? I tested with Urdu text and locale and it exhibits the same behavior (rendering different fonts in different weights).

Reply:
`zh` is a little special because of the "LG Smart UI TC" font, which includes latin characters. Because of this they were clobbering the standard font when it was present. I did not exhaustively check every font for latin or other overlapping codepoints (other than Japanese). When testing Urdu, which font weights did you see multiple typefaces for and what text were you using? I'd like to repro locally and look into the numerous other questions I have.

Reply:
Why don't we set weights in css for all single weight fonts like Urdu? We have still many single weight fonts
(https://wiki.lgsvl.com/pages/viewpage.action?title=Fonts+for+webOS+Jheri+Curl&spaceKey=webOSDocs).
I've tested this PR in rc4, but I can still see this bug in Tamil(`ta-IN`) too. We need to test all locale that use single weight font.

Reply:
We do set multiple font weights for Urdu. This is an "outdated" comment thread.

You probably shouldn't be linking to internal company web pages in a public place.

It's possible we need to add explicit font-weight assignments to other language fonts too, like Tamil, but I'd like to be deliberate about which ones we add it to, since the behavior doesn't seem to be consistent across all of the locale fonts. Comprehensive testing sounds like a good idea. We just need a time-efficient means of doing that. Open to suggestions.